src/app/SharpRaven/Data/JsonPacket.cs

```diff
@@ -396,5 +397,10 @@ private void Initialize(Exception exception)
                 }
             }
         }
+
+        public override int GetHashCode()
+        {
+            return this.ToString().GetHashCode()*17;
```

Comment:
I noticed that this is used as the file name of `JsonPacket` when serialised to disk, which sounds like a bad idea since `GetHashCode` isn't guaranteed to be unique. Using `Guid.NewGuid` or some other way to generate a guaranteed unique, but file-system friendly name, would be better.